frontend/src/scenes/session-recordings/errors/sessionRecordingErrorsLogic.ts

```diff
@@ -19,7 +25,18 @@ export const sessionRecordingErrorsLogic = kea<sessionRecordingErrorsLogicType>(
             },
         ],
     })),
+    listeners(() => ({
+        createPlaylist: async ({ name, sessionIds }) => {
+            const playlist = await createPlaylist({ name: name })
 
+            if (playlist) {
+                await Promise.all(
+                    sessionIds.map((sessionId) => api.recordings.addRecordingToPlaylist(playlist.short_id, sessionId))
```

Comment:
🤔 we should maybe limit the size of the recordings we will add to the playlist here

e.g. if there are 4500 items in a cluster do we want to add them all to a playlist

and do we want to make 4.5k API calls?

i'd definitely do at least one of limiting the list or amending the API to take a list of ids

Reply:
would be possible to paginate? so we query the next 100s if the user goes to the next 100

Reply:
Good call. In this instance we were protected by the fact that only 10 samples are returned from the backend: https://github.com/PostHog/posthog/pull/21037/files#diff-18f37520b0e5b0957994182f1c1b6b582893c2d04e87b88e75e967cd6fcc64abR84. But no harm slicing the array on the frontend too. Pushed a new commit

---

@marandaneto think that is the way to go once we have split out static / dynamic playlists (next on my list but the refactor is taking longer than expected)